command/scaling_policy_info.go

```diff
@@ -54,6 +57,21 @@ func (s *ScalingPolicyInfoCommand) AutocompleteFlags() complete.Flags {
 		})
 }
 
+func (c *ScalingPolicyInfoCommand) AutocompleteArgs() complete.Predictor {
+	return complete.PredictFunc(func(a complete.Args) []string {
+		client, err := c.Meta.Client()
+		if err != nil {
+			return nil
+		}
+
+		resp, _, err := client.Search().PrefixSearch(a.Last, contexts.ScalingPolicies, nil)
+		if err != nil {
+			return []string{}
+		}
+		return resp.Matches[contexts.ScalingPolicies]
+	})
+}
+
```

Comment:
I am not sure if this was how it supposed to work, but it doesn't seem to match the behaviour for `nomad deployment status` for example:

```shellsession
$ nomad deployment status <TAB>
050fe10a-2424-9c4d-7395-0ce15bea2b4e  2e85e08d-c247-f44b-d935-15b15adf45d0  3e874047-3565-be07-61b7-c1606360076c  9cdb46b3-a4c5-89ab-15e7-d492e23cf884  af0e331f-00ae-c9c1-df57-366c5778e35a  b0299e46-dc61-b37c-19ae-37fe58c8faf3  c00bd7de-fc17-ae3b-2711-4b6319cd51c0
212be987-88c6-7d32-58f3-4dbae98997db  320793f0-d19d-20f2-0f4a-08d29de8dff9  6f19ecd3-8ba8-11ac-234d-43b0878474d4  9e195d24-0605-400a-5b7e-f4794148f8d5  afcba73a-326a-e487-18d4-3020b489aae4  be444b5c-6e15-2286-ded0-91558018e4ee
```

With `nomad scaling policy info <TAB>` I don't get any suggestion.

Reply:
i do... although i have to hit tab twice (same for `deployment status`): 
```
$ nomad scaling policy info <TAB>
132a8a8c-b45c-e72b-8054-3f4a4aecd8d5  20c79cfb-96c3-33cb-d0d9-fe136bb6c69e
17affc7e-c5ce-c59b-575e-d1502c3e588d  4e99c361-60ea-2f92-cfaa-e8e9df581340
```

Reply:
Oh, I probably installed it wrong then 😅